src/components/Carousel/index.js

```diff
@@ -0,0 +1,3 @@
+import Carousel from './Carousel';
+
```

Comment:
can we export carousel and import from the carousel file? Also it would be nice to add an example carousel into homepage.js (you may need to update your branch) 

Use images from https://unsplash.com/ for previews since it'll look more like what we're going for

Reply:
Any images?

Reply:
they're placeholders, doesn't matter.